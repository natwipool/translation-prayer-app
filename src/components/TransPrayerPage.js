import React from 'react';
import TransPrayerLists from './TransPrayerLists'

const TransPrayerPage = () => (
  <div>
    <TransPrayerLists />
  </div>
);

export default TransPrayerPage;