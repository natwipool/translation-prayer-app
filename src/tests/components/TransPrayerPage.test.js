import React from 'react';
import { shallow } from 'enzyme';
import TransPrayerPage from '../../components/TransPrayerPage';

test('should render TransPrayerPage correctly', () => {
  const wrapper = shallow(<TransPrayerPage />);
  expect(wrapper).toMatchSnapshot();
});