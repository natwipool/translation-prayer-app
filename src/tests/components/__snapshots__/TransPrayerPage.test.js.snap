// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TransPrayerPage correctly 1`] = `
<div>
  <Connect(TransPrayerLists) />
</div>
`;
